homeassistant/helpers/script.py

```diff
@@ -187,9 +189,11 @@ def async_continue_stop(command=None):
 
     try:
         yield trace_element
-    except _StopScript as ex:
+    except _AbortScript as ex:
         trace_element.set_error(ex.__cause__ or ex)
         raise ex
+    except _StopScript as ex:
```

Comment:
Is this really needed?
I mean, we're catching the exception and raising it again without doing anything with the exception...

Reply:
Yes, else it is caught by the one below...

Reply:
Ah yes, I didn't realize that the next block caught all exceptions.